components/library/libvncserver/Makefile

```diff
@@ -51,12 +47,14 @@ CMAKE_OPTIONS += -DJPEG_INCLUDE_DIR=$(JPEG_INCDIR)
 CMAKE_OPTIONS += -DJPEG_LIBRARY=$(JPEG_LIBDIR)/libjpeg.so
```

Comment:
You should be able to remove the jpeg-related flags completely.
It should even be possible to remove the CMAKE_OPTION related to jpeg.